clang/include/clang/Driver/ToolChain.h

```diff
@@ -619,7 +619,13 @@ class ToolChain {
   virtual bool SupportsEmbeddedBitcode() const { return false; }
 
   /// getThreadModel() - Which thread model does this target use?
-  virtual std::string getThreadModel() const { return "posix"; }
+  virtual std::string getThreadModel() const {
+#ifdef _WIN32
+    return "win32";
```

Comment:
I think this rather should check the target OS, than to decide based on the host OS that the binary is running on; in particular, the test you're modifying uses an explicit ARM linux target.

Reply:
Oh, you are right. This change as is is pretty dumb right now.